src/main/java/org/isf/accounting/gui/PatientBillEdit.java

```diff
@@ -290,168 +280,219 @@ public void patientSelected(Patient patient) {
 	private static final int PAYMENT_HEIGHT = 150;
 	private static final int BALANCE_HEIGHT = 20;
 	private static final int BUTTON_HEIGHT = 25;
+	private static final Dimension PATIENT_DIMENSION = new Dimension(300, 20);
+	private static final Dimension LABELS_DIMENSION = new Dimension(60, 20);
+	private static final Dimension USER_DIMENSION = new Dimension(220, 20);
+	private static final Dimension WARD_DIMENSION = new Dimension(195, 20);
+	private static final Dimension TOTAL_TABLE_SIZE = new Dimension(PANEL_WIDTH, TOTAL_HEIGHT);
+	private static final Dimension BIGTOTAL_TABLE_SIZE = new Dimension(PANEL_WIDTH, BIG_TOTAL_HEIGHT);
+	private static final Dimension BALANCE_TABLE_SIZE = new Dimension(PANEL_WIDTH, BALANCE_HEIGHT);
+	private static final Dimension BUTTON_ITEM_SIZE = new Dimension(BUTTON_WIDTH_BILL, BUTTON_HEIGHT);
+	private static final Dimension BUTTON_PAYMENT_SIZE = new Dimension(BUTTON_WIDTH_PAYMENT, BUTTON_HEIGHT);
+	private static final Dimension BUTTON_ACTION_SIZE = new Dimension(BUTTON_WIDTH, BUTTON_HEIGHT);
 
 	private BigDecimal total = new BigDecimal(0);
 	private BigDecimal bigTotal = new BigDecimal(0);
 	private BigDecimal balance = new BigDecimal(0);
-	private int billID;
-	private PriceList listSelected;
 	private boolean insert;
 	private boolean modified = false;
```

Comment:
assigning `false` is redundant; it is `false` by default.
```suggestion
	private boolean modified;
```